index.html

```diff
@@ -1,173 +1,273 @@
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
             <ul class="stack-container">
-                <li class="border-right padding-right">css</li>
-                <li class="border-right padding-right">html</li>          
-                <li class="border-right padding-right">bootstrap</li>         
-                <li class="padding-right">Ruby</li>
+              <li class="border-right padding-right">css</li>
+              <li class="border-right padding-right">html</li>
+              <li class="border-right padding-right">bootstrap</li>
+              <li class="padding-right">Ruby</li>
             </ul>
             <button type="button" class="project-button">See Project</button>
-
-            </div>            
-        
-        </section> 
-    </section>
-    <section class="form-container">
+          </div>
+        </section>
+      </section>
+      <section class="form-container" id="contact">
         <h3 class="mobH">CONTACT US</h3>
         <h3 class="deskH">Let`s chat!</h3>
-        <P>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </P>
-        <form action="https://formspree.io/f/xdoylybq" method="POST">
-            <label for="name"></label>
-            <input type="text" id="name" name="Full name" placeholder="Full Name" required>
-            <input type="email" id="mail" name="_replyto" placeholder="jeremy@gmail.com"  required>
-            <label for="msg"></label>
-            <textarea name="message" id="msg" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
-            <button class="project-button" type="submit">Submit</button>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form action="https://formspree.io/f/xdoylybq" method="POST" id="formV">
+          <label for="name"></label>
+          <input
+            type="text"
+            id="name"
+            name="Full name"
+            placeholder="Full Name"
+            required
+          />
+          <input
+            type="email"
+            id="mail"
+            name="_replyto"
+            placeholder="jeremy@gmail.com"
+            required
+          />
+          <label for="msg"></label>
+          <textarea
+            name="message"
+            id="msg"
+            cols="30"
+            rows="10"
+            placeholder="Enter text here"
+            maxlength="500"
+            required
+          ></textarea>
+          <button class="project-button" type="submit">Submit</button>
```

Comment:
- [ ] The error message should show up close to the "submit" button. All errors in the input fields like an empty field or syntax errors should be raised in a paragraph close to the submission button. This is because some browsers won't raise an error by default so it's important to add error messages to make your form works well in every browser.